### PR TITLE
Actor view create update

### PR DESCRIPTION
**Pull request by laura-wenning**
Adds code for handling the creation, viewing, and potential updating of an actor document. The updating is currently unused due to issues from the

### CODE REVIEW

File: src/components/sheets/ActorSheet.tsx [hunk 1]
```diff
@@ -15,23 +15,35 @@ interface ActorSheetProps {
...
   return (
     <Formik
       initialValues={{}}
+      
       onSubmit={console.log}
     >
-      <>
-        {sheetElements}
-      </>
+      {(formikProps: FormikProps<Record<string, unknown>>) => {
```
Comment (laura-wenning) [inline src/components/sheets/ActorSheet.tsx:25]: Revert these changes; this breaks the actor sheet

File: src/pages/dev/actors/[ref]/index.tsx [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+import { ActorSheet } from "components/sheets/ActorSheet";
+import { ActorData } from "controllers/data/ActorData";
+import { ActorSheetData } from "controllers/data/ActorSheetData";
+import { CampaignData } from "controllers/data/CampaignData";
+import { RulesetData } from "controllers/data/RulesetData";
+import { observer } from "mobx-react-lite";
+import { useRouter } from "next/router";
+import React from "react";
+
+function ActorView() {
```
Comment (laura-wenning) [inline src/pages/dev/actors/[ref]/index.tsx:12]: Comment

File: src/pages/dev/actors/[ref]/index.tsx [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+  const router = useRouter();
+  const ref = router.query.ref as string;
+
+  React.useEffect(() => {
+    ActorData.load(ref);
+  }, [ref]);
+
+  const actor = ActorData.get(ref);
+  React.useEffect(() => {
```
Comment (laura-wenning) [inline src/pages/dev/actors/[ref]/index.tsx:21]: Comment

File: src/pages/dev/actors/[ref]/index.tsx [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+
+  const actorSheet = ActorSheetData.get(actor?.actorSheet?.ref);
+  React.useEffect(() => {
+    if (!actorSheet || !actorSheet.ref) { return; }
+    ActorSheetData.loadSheet(actorSheet.ref);
+  }, [actorSheet]);
+
+  if (!actor) { return <></>; }
+
+  function onSubmit(values: Record<string, unknown>) {
```
Comment (laura-wenning) [inline src/pages/dev/actors/[ref]/index.tsx:36]: Fix, comment, and use a save button for now

File: src/server/logic/ActorLogic.ts [hunk 5]
```diff
@@ -22,4 +55,46 @@ class $ActorLogic {
   }
 }
 
+function randomActorName() {
```
Comment (laura-wenning) [inline src/server/logic/ActorLogic.ts:58]: Comment

**Review by lucyawrey — APPROVED** [review]
<3
